WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -264,18 +264,25 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
             return
         }
 
-        /// Jetpack is required. Present an error if we don't detect a valid installation for a self-hosted site.
-        if let site = currentSelfHostedSite,
-            site.url == wpcomLogin.siteURL, !site.hasValidJetpack {
-            let viewModel = JetpackErrorViewModel(siteURL: wpcomLogin.siteURL)
-            let installJetpackUI = ULErrorViewController(viewModel: viewModel)
-            return navigationController.show(installJetpackUI, sender: nil)
-        }
```

Comment:
This part is moved down below to make sure that when this method is called again after Jetpack setup completes, we'll no longer display the Jetpack error screen.

Reply:
On a different note, could you please explain why you have to move this inside `guard matcher.match(originalURL: wpcomLogin.siteURL)` part to avoid showing the Jetpack error screen again?

I am asking this because I have to show the no Jetpack error for self-hosted sites in a future PR. 😄 

FYI, those self-hosted sites will have only .org credentials an,d they will not have WPCOM credentials.
i.e. 
`AuthenticatorCredentials` - `wpcom` will be `nil`
and 
`AuthenticatorCredentials` - `wporg` will be non nil.

And, I plan to take the `siteURL` from `wporg` credentials.